.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
No reason for this to be here. We should allow warnings when there are significant number of changes in a single commit to enforce better Git practices.  This will be removed.